source/configure/plugins-configuration-settings.rst

```diff
@@ -739,6 +739,32 @@ Allow screen sharing
 | Changing this setting requires a plugin restart to take effect.        |                                           |
 +------------------------------------------------------------------------+-------------------------------------------+
 
+.. config:setting:: plugins-callsenablesimulcast
+  :displayname: Enable simulcast for screen sharing (Experimental) (Plugins - Calls)
+  :systemconsole: Plugins > Calls
+  :configjson:
```

Comment:
If there isn't a ``config.json`` equivalent, please populate the ``configjson`` and ``environment`` fields with ``N/A``.